README.md

```diff
@@ -5,6 +5,10 @@ with bazel.
 
 ## Usage
 
+### Prerequisites
+
+Please make sure you have at least Python 3.12, `tar`, `zstd`, and `xz-utils` installed.
```

Comment:
i attempted to make sure it worked with <3.12 as long as you installed, but haven't stress tested